retrieve_oa_files.py

```diff
@@ -292,7 +291,9 @@ def postFromS3ToJSon(obj):
         docid = Util.doc_id(obj.key)
         objdata = obj.get()
         jsontext = objdata['Body'].read()
-        jsontext = jsontext.replace(b'\n', b'')
```

Comment:
@vladistan Instead of removing this line for now, can we just replace it with the following:
`jsontext = jsontext.replace(b'\n', b' \n ')`?